Trackers/iou_pred_tracker/iou_pred.py

```diff
@@ -95,25 +97,17 @@ def update(self, detections):
             for i in range(0, len(detections)):
                 self.add_track(self.nextID, detections[i])
             return self.tracks
-        tracks = []
         predicts = []
         for trk in self.tracks:
-            tracks.append(trk.bbox)
             if(trk.hits > self.min_hits):
-                predict_bbox = trk.iou_predict(trk.bbox, trk.history)
+                predict_bbox = trk.iou_predict()
                 predicts.append(predict_bbox)
             else:
+                trk.history.append(trk.bbox)
                 predicts.append(list(trk.bbox))
```

Comment:
Handle this logic in the `iou_predict` itself

Reply:
We cannot call `iou_predict` function till we reach the min_hits which is 3, so i guess we cannot implement the logic of else statement in `iou_predict`

Reply:
I have made the required changes in my branch, sorry for delay.